src/bin/medic-conf.js

```diff
@@ -30,6 +30,13 @@ switch(args[0]) {
   default:          log.level = log.LEVEL_INFO;
 }
 
+const acceptSSL = '--accept-ssl';
```

Comment:
The way we parge args here is super order specific (to the point of being broken in my mind). 
I think this code needs to be under the `--no-check` code lower down in this file or else it is inconsistent with the documented order you have in help docs.

Reply:
Agree. There are great libraries to deal with this kind of arg parsing that are probably better at this point than doing it ourselves.